packit_service/worker/handlers/abstract.py

```diff
@@ -209,6 +211,49 @@ def get_packit_commands_from_comment(
     return []
 
 
+class CeleryTask:
+    def __init__(self, task: Task):
+        self.task = task
+
+    @property
+    def retries(self):
+        return self.task.request.retries
+
+    def is_last_try(self) -> bool:
+        """
+        Returns True if the current celery task is run for the last try.
+        More info about retries can be found here:
+        https://celeryproject.readthedocs.io/en/latest/userguide/tasks.html#retrying
+        """
+        return self.retries == self.get_retry_limit()
+
+    def get_retry_limit(self) -> int:
+        """
+        Returns the limit of the celery task retries.
+        (Packit uses this env.var. in HandlerTaskWithRetry base class
+        to set `max_retries` in `retry_kwargs`.)
+        """
+        return int(getenv("CELERY_RETRY_LIMIT", DEFAULT_RETRY_LIMIT))
+
+    def retry(self, ex: Exception, delay: Optional[int] = None) -> None:
+        """
+        Retries the celery task.
+        Argument `throw` is set to False to not retry
+        the task also because of the `autoretry_for` mechanism.
+
+        More info about retries can be found here:
+        https://celeryproject.readthedocs.io/en/latest/userguide/tasks.html#retrying
+
+        :param ex: Exception needs to be specified.
```

Comment:
Since refactoring is being done, could we make the docstring consistent with the rest of the code? Most methods I see use the `Args:` notation. Also a suggestion for rewording:

```
Args:
    ex: Exception which caused the retry (will be logged).
    delay: Number of seconds the task will wait before being run again.
```